tests/PHPStan/Rules/Exceptions/CatchWithUnthrownExceptionRuleTest.php

```diff
@@ -12,11 +13,21 @@
 class CatchWithUnthrownExceptionRuleTest extends RuleTestCase
 {
 
+	private static bool $detectDeadMultiCatch = true;
+
 	protected function getRule(): Rule
 	{
 		return new CatchWithUnthrownExceptionRule();
 	}
 
+	public static function getAdditionalConfigFiles(): array
+	{
+		return array_merge(
+			parent::getAdditionalConfigFiles(),
+			!self::$detectDeadMultiCatch ? [__DIR__ . '/disable-detect-multi-catch.neon'] : [],
```

Comment:
Should be two separate test cases instead of switching the property. I haven't tested if this works, and the convention is to have two separate test cases... You can't be sure when `getAdditionalConfigFiles` will be called first and thus it can lead to unexpected behaviour.